test/e2e-go/restAPI/other/misc_test.go

```diff
@@ -58,12 +58,22 @@ func TestDisabledAPIConfig(t *testing.T) {
 	assert.True(t, os.IsNotExist(err))
 
 	// check public api works without a token
-	testClient.WaitForRound(1)
+	url, err := localFixture.NC.ServerURL()
+	a.NoError(err)
+	testClient := client.MakeRestClient(url, "") // empty token
+
+	_, err = testClient.WaitForBlock(1)
+	assert.NoError(t, err)
 	_, err = testClient.Block(1)
 	assert.NoError(t, err)
+	_, err = testClient.Status()
+	a.NoError(err)
+
 	// check admin api works with the generated token
-	_, err = testClient.GetParticipationKeys()
+	adminClient := localFixture.LibGoalClient
```

Comment:
NIT: you have the libgoalClient variable on line 42 that appears to be the same thing.

Reply:
applied